.readme-partials.yaml

```diff
@@ -134,3 +134,50 @@ custom_content: |
+  the [structured logging Json format](https://cloud.google.com/logging/docs/structured-logging#special-payload-fields).
+  To do this, add `com.google.cloud.logging.LoggingHandler.redirectToStdout=true` to the logger configuration file.
+  You can use this configuration when running applications in Google Cloud managed environments such as AppEngine, Cloud Run,
+  Cloud Function or GKE. The logger agent installed on these environments can capture STDOUT and ingest it into Cloud Logging.
+  The agent can parse structured logs printed to STDOUT and capture additional log metadata beside the log payload.
+  The parsed information includes severity, source location, user labels, http request and tracing information.
+
+  #### Auto-population of log entrys' metadata
+
+  LogEntry object metadata information such as [monitored resource](https://cloud.google.com/logging/docs/reference/v2/rest/v2/MonitoredResource), 
+  [Http request](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#HttpRequest) or 
+  [source location](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#LogEntrySourceLocation)
+  are automatically populated with information that the library retrieves from the execution context.
+  The library populates only empty (set to `null`) LogEntry fields.
+  This behavior can be opted out by calling `LoggingOptions.Builder.setAutoPopulateMetadata(false)`.
+  Cloud Logging handler can be configured to opt-out automatic population of the metadata via the config file by
+  setting `com.google.cloud.logging.LoggingHandler.autoPopulateMetadata` configuration to `false`.
+
+  The auto-population logic populates source location _only_ for log entries with `Severity.DEBUG` severity.
+  The execution context of the Http request and tracing information is maintained by `ContextHandler` class.
+  The context is managed in the scope of the thread.
+  The library provides two methods to update the context:
+  1. Using [servlet initializer](https://github.com/googleapis/java-logging-servlet-initializer).
+  If your application uses a Web server based on Jakarta servlets (e.g. Jetty or Tomcat), you can add the servlet initializer
+  package to your WAR. The package implements a service provider interface (SPI) for
+  [javax.servlet.ServletContainerInitializer](https://docs.oracle.com/javaee/6/api/javax/servlet/ServletContainerInitializer.html)
+  and filters all servlet requests to automatically capture the execution context of the servlet request and store it using
+  `ContextHandler` class. The stored `Context` class instances are used to populate Http request and tracing information.
+  If you use Maven, to use the servlet initializer add the following dependency to your BOM:
+
+      ```xml
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-logging-servlet-initializer</artifactId>
+      </dependency>      
+      ```
+
+  1. Manually set the context.
```

Comment:
Can you please make this option to be #1 (e.g. default option)? And move servlet initializer to be #2?

Reply:
I will remove the numbers to avoid implying preferences or order for the options. By your request I will place the manual option to be the first in the list.